### ISSUE TITLE
Use Wii U Icons

### ISSUE THREAD

**Issue by supergeorg** [JosefNemec/Playnite#314]
In every Wii U Game Folder, there is a "/meta/iconTex.tga" file.

It would be nice if it is recognized on scanning emulator games. (Although Playnite has to add .tga support to use it or convert it somehow)

**Comment by JosefNemec** [comment]
That might be a bit tricky because WPF (UI framework used by Playnite) doesn't support TGA format out of the box.

**Comment by supergeorg** [comment]
Yeah, but icons get cached in the library database, so maybe it would be possible to convert them to a readable format and store them there. At least this was recommended by Microsoft here: [https://social.msdn.microsoft.com/Forums/de-DE/22da10c0-1fa1-4ae2-8800-a64b8bd464ef/darstellen-von-tga-in-wpf?forum=wpfde](url)

**Comment by JosefNemec** [comment]
Depends on #279
